pom.xml

```diff
@@ -1237,11 +1238,55 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <!-- https://mvnrepository.com/artifact/org.openclover/clover-maven-plugin -->
+            <dependency>
+                <groupId>org.openclover</groupId>
+                <artifactId>clover-maven-plugin</artifactId>
+                <version>${version.org.openclover}</version>
+            </dependency>
+
         </dependencies>
     </dependencyManagement>
 
     <profiles>
         <profile>
+<!--To generate a report on test coverage run
```

Comment:
Looks like the indentation just needs to be adjusted here but this is looking good in general!